### DISCUSSION TITLE
Can Mia-Platform be used to build KubeEdge applications?

### DISCUSSION THREAD

**Discussion by calairoh** [mia-platform/community#232] [Q&A] [answered]
### Description

We know that Mia-Platform allows to create cloud-native application based on K8s.
My main question is: **Mia-Platform can be used to build applications which will be deployed on edges?**

I try to explain with an example use-case. Let us assume that we want to create a simple application with a frontend, a backend and a storage. Let's also assume that we want to deploy this application not only in cloud but also to edge devices, such as a local PC.
In my knowledge, this is possible using a specific distro of k8s built for edge computing, that is KubeEdge.

However, I have some technical doubts that I need to clarify. In particular:
- **Is it possible to use the configuration created by the console to easily deploy such an application to a device?**
- **How can the deployment be performed in the practice?**

**Comment by davidebianchi** [comment, marked as answer]
Hi, I never used kube edge, so I can only try to answer to your question but it should be checked. In the past we made some test with kube edge, but I don't remember how much it was possible to use Console.

* Once configured I suppose it should be possible to use the configuration created by Console to deploy an application since it should support Kubernetes resources.
* I don't know how deployment can be performed, I suppose it is a standard K8s deploy maybe with some node affinity to set in the application. In this case, it could be possible to try using kustomize maybe. 

Do you have some specific use case?
